### PR TITLE
Add Basic Support for Allworx VoIP Systems

### PR DESCRIPTION
**Pull request by VanillaNinjaD**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: includes/polling/os/allworx.inc.php [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?php
+$hardware = $device['sysDescr'];
+$version = snmp_get($device, '1.3.6.1.2.1.27.1.1.4.1', '-OQv', '', '');
```
Comment (laf) [inline includes/polling/os/allworx.inc.php]: You should add a license header to the files you add.

Can you please convert these two snmp_gets into multi gets - check other files for examples.

File: includes/definitions/allworx.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+os: allworx
```
Comment (laf) [inline includes/definitions/allworx.yaml]: allworx is the vendor name, do they not have an OS name these products run or a group name for the product line?

File: html/images/os/allworx.svg [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (laf) [inline html/images/os/allworx.svg]: You need to remove the width/height + compress the svg.

Use https://jakearchibald.github.io/svgomg/ to do this.

File: includes/definitions/allworx.yaml (hunk 2 above)
Comment (VanillaNinjaD) [reply includes/definitions/allworx.yaml]: The have both the X (legacy) series and the Connect (current) series of telephone switches but they don't have an OS name and the software is unified between the models. I am happy to change it but I don't know what we'd change it to.

Any suggestions?

Comment (VanillaNinjaD) [reply includes/definitions/allworx.yaml]: Since Allworx makes a line of Layer 3 smart switches I forgot about, I'll rename this OS to allworx_voip

**Comment by VanillaNinjaD** [comment]
@laf Here is my best effort!

**Comment by laf** [comment]
@VanillaNinjaD I've made a few teaks. Once it's passed tests, we'll merge in.

**Pull request merged**